test/spec/algoliasearch.helper/pendingSearch.js

```diff
@@ -5,35 +5,6 @@ var algoliaSearch = require('algoliasearch');
 
 var algoliasearchHelper = require('../../../index');
 
-test('When searchOnce with callback, hasPendingRequests is true', function(t) {
```

Comment:
I removed this test because `search()` with callbacks doesn't exist anymore. Let me know if I'm wrong @bobylito.

Reply:
SearchOnce should continue to accept callbacks, even if we've changed the internal implementation. Therefore  we should not remove this test, I think.

Reply:
Yep, sorry I got confused!